### DISCUSSION TITLE
defmain replacement?

### DISCUSSION THREAD

**Discussion by nkabir** [hylang/hy#2255] [Q&A] [answered]
I am migrating my code from 0.20 to 1.0a4 but have been unable to find the replacement for `defmain`. 

https://docs.hylang.org/en/stable/language/api.html#defmain

Is there an example of an executable script for hy? Thank you for your excellent work!

**Comment by nkabir** [comment, marked as answer]
I found it. Google did not pick up the new references but searching within the hyrule project returns the new implementation:

https://hyrule.readthedocs.io/en/master/index.html?highlight=defmain#hyrule.control.defmain

**Reply by Kodiologist** [reply]
Hy's NEWS file is helpful for this sort of thing.
